src/pages/workspace/expensifyCard/WorkspaceExpensifyCardPageEmptyState.tsx

```diff
@@ -40,6 +43,23 @@ function WorkspaceExpensifyCardPageEmptyState({route, policy}: WorkspaceExpensif
     const styles = useThemeStyles();
     const theme = useTheme();
     const {shouldUseNarrowLayout} = useResponsiveLayout();
+    const [bankAccountList] = useOnyx(ONYXKEYS.BANK_ACCOUNT_LIST);
+    const [reimbursementAccount] = useOnyx(ONYXKEYS.REIMBURSEMENT_ACCOUNT);
+
+    const reimbursementAccountStatus = reimbursementAccount?.achData?.state ?? '';
+    const isSetupUnfinished =
+        isEmptyObject(bankAccountList) &&
+        (reimbursementAccountStatus === CONST.BANK_ACCOUNT.STATE.VERIFYING ||
+            reimbursementAccountStatus === CONST.BANK_ACCOUNT.STATE.SETUP ||
+            reimbursementAccountStatus === CONST.BANK_ACCOUNT.STATE.VALIDATING);
```

Comment:
I think you can instead check if the status is defined and its not OPEN

Reply:
will be held in https://github.com/Expensify/App/pull/46564